platform/android/java_godot_io_wrapper.cpp

```diff
@@ -53,6 +53,7 @@ GodotIOJavaWrapper::GodotIOJavaWrapper(JNIEnv *p_env, jobject p_godot_io_instanc
 		_get_locale = p_env->GetMethodID(cls, "getLocale", "()Ljava/lang/String;");
 		_get_model = p_env->GetMethodID(cls, "getModel", "()Ljava/lang/String;");
 		_get_screen_DPI = p_env->GetMethodID(cls, "getScreenDPI", "()I");
+		_get_screen_refresh_rate = p_env->GetMethodID(cls, "getScreenRefreshRate", "(D)D");
```

Comment:
I'm not seeing the matching java method this is supposed to link to. Was it added in another PR?

Reply:
Good catch, I forgot to port it from `master` :slightly_smiling_face: 

Fixed.